CMakeLists.txt

```diff
@@ -103,7 +103,8 @@ if (WIN32)
     set(Boost_THREADAPI  win32)
 else()
     set(BOOST_INCLUDEDIR "/usr/local/include/boost")
-    set(Boost_LIBRARYDIR "/usr/local/lib/")
+    set(BOOST_LIBRARYDIR "/usr/local/lib")
+
```

Comment:
Remove empty line please.

Reply:
https://cmake.org/cmake/help/latest/module/FindBoost.html
BOOST_LIBRARYDIR
Preferred library directory e.g. <prefix>/lib.


Maybe we shoudl remove `/lib` and keep only `set(BOOST_LIBRARYDIR "/usr/local")`

Reply:
The <prefix> is set by `BOOST_ROOT`. I think "/usr/local/lib" is the correct default value for `BOOST_LIBRARYDIR`.